subcommand/lifecycle-sidecar/command_test.go

```diff
@@ -84,6 +89,115 @@ func TestRun_ConsulBinaryMissing(t *testing.T) {
 	require.Contains(t, ui.ErrorWriter.String(), "-consul-binary \"/not/a/valid/path\" not found")
 }
 
+// Test that we register the services.
+func TestRun_ServicesRegistration(t *testing.T) {
+	t.Parallel()
+	tmpDir, err := ioutil.TempDir("", "")
+	require.NoError(t, err)
+	defer func() { os.RemoveAll(tmpDir) }()
```

Comment:
Does this need the `func` wrapper?

Reply:
I don't think it does, good catch. I'll remove it.